scripts/v.dissolve/v.dissolve.py

```diff
@@ -33,36 +33,339 @@
 # %end
 # %option G_OPT_V_OUTPUT
 # %end
+# %option G_OPT_DB_COLUMN
+# % key: aggregate_column
+# % description: Name of attribute columns to get aggregate statistics for
+# % multiple: yes
+# %end
+# %option
+# % key: aggregate_method
+# % label: Aggregate statistics method
+# % description: Default is all available basic statistics
+# % multiple: yes
+# %end
+# %option G_OPT_DB_COLUMN
+# % key: stats_column
+# % description: New attribute column name for aggregate statistics results
+# % description: Defaults to aggregate column name and statistics name
+# % multiple: yes
+# %end
+# %option
+# % key: aggregate_backend
+# % label: Aggregate statistics method
+# % description: Default is all available basic statistics
```

Comment:
Default is auto-detection based on aggregate_method.

Reply:
Not it says: `Default is sql unless the methods are for univar`

Reply:
Now it says the right thing.